arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcDataOperation.java

```diff
@@ -141,11 +141,11 @@ private EcDataOperation(
         "EcDataOperation should only be called with nonempty call rightPaddedCallData");
     final int paddedCallDataLength =
         switch (precompileFlag) {
-          case PRC_ECRECOVER -> 128;
-          case PRC_ECADD -> 128;
-          case PRC_ECMUL -> 96;
+          case PRC_ECRECOVER -> TOTAL_SIZE_ECRECOVER_DATA;
+          case PRC_ECADD -> TOTAL_SIZE_ECADD_DATA;
+          case PRC_ECMUL -> TOTAL_SIZE_ECMUL_DATA;
           case PRC_ECPAIRING -> {
-            Preconditions.checkArgument(callDataSize % 192 == 0);
+            Preconditions.checkArgument(callDataSize % TOTAL_SIZE_ECPAIRING_DATA_MIN == 0);
```

Comment:
The use of `MIN` isn't right. It's the size of a pair of points (small + large) for `ECPAIRING` and every invokation must have `call_data_size` that is a multiple of that. In particular it's neither a `TOTAL_SIZE` nor a `MIN` (empty call data is allowed, too, though it won't transition through the present module.)

Reply:
Maybe
```rust
ECPAIRING_BYTE_SIZE_OF_PAIR_OF_POINTS
```

Reply:
And obviously these constants would have to be declared in the LISP. The consumers of these constants would be
- HUB
- OOB
- ... maybe other modules, too ?

Reply:
they are already part of the lisp and the specs